### DISCUSSION TITLE
running multiple seleniumbase test cases from within a python wrapper, collecting the output for further use/interpretation

### DISCUSSION THREAD

**Discussion by nico599** [seleniumbase/SeleniumBase#1661] [Q&A] [answered]
We are wondering if it is possible to develop a python wrapper that can run multiple selenium tests and keep the output within python for more simple interpretation and error handling.

e.g:
for testFile in /path/to/folder
try 
seleniumbase(testFile), collect result, latency, errormessage(if fail) into variable.

**Comment by mdmintz** [comment, marked as answer]
The XML report option is probably the best way for simplified results that you can use later. Easy to read the file and parse the data. For that, use --junit-xml=report.xml to get the xml report. Jenkins can also use this file to display results better.

**Reply by nico599** [reply]
We'll give it a go, cheers. Great work on this project by the way. Fantastically simple and elegant.

**Reply by mdmintz** [reply]
Thank you!
